android/engine/src/main/java/org/smartregister/fhircore/engine/data/local/register/dao/DefaultPatientRegisterDao.kt

```diff
@@ -90,7 +91,7 @@ constructor(
         identifier = patient.identifierFirstRep.value,
         address = patient.extractAge(),
         gender = patient.gender,
-        birthdate = patient.birthDate,
+        birthdate = patient.birthDate ?: Date(),
```

Comment:
Can we make this nullable instead? Defaulting to today's date would be incorrect.

Reply:
its reverted, but couldn't push yesterday, doing shortly, 
I added to bypass crash as its been crashing on most of patient details, (client list to profile screen)